numpy/_core/tests/test_multiarray.py

```diff
@@ -8285,7 +8285,7 @@ def test_padded_struct_array(self):
         self._check_roundtrip(x3)
 
     @pytest.mark.valgrind_error(reason="leaks buffer info cache temporarily.")
-    def test_relaxed_strides(self, c=np.ones((1, 10, 10), dtype='i8')):
+    def test_relaxed_strides(self, c=np.ones((1, 10, 10), dtype='i8')):  # noqa: B008
```

Comment:
Fun fact: arrays as function parameter defaults were causing mypy's `stubtest` to crash on `mypy<1.15`: https://github.com/python/mypy/issues/18343#issuecomment-2564314519

So just to be safe; maybe use `None` as "default default", and do a `c = np.ones(...) if c is None else c` in the function body?

Reply:
I thought the test is executed only once, but yes, that would be safer in the long term and cleaner.